server/src/db/index.js

```diff
@@ -0,0 +1,7 @@
+import mongoose from "mongoose";
+// console.log("Mongo URL: ", process.env.MONGODB_URL);
+mongoose.connect(process.env.MONGODB_URL, {
```

Comment:
What will happen if it throws the error? Also, does this will start after the server is started or before? If after then what will the user will see and if s/he requests the server?